src/pkg/path/path.go

```diff
@@ -127,6 +129,25 @@ type RestorePaths struct {
 	RestorePath Path
 }
 
+type FileSystem interface {
```

Comment:
I guess you created it for use in tests. Just for completeness, we call add `os.Stat` as well here I guess. The test implementation can be just a `panic` if we are not using it.